detection-rules/attachment_callback_scam.yml

```
@@ -0,0 +1,31 @@
+name: "Attachment: Callback scam from first-time sender"
+description: |
+    Detects callback scams by analyzing text within images of receipts or invoices from first time senders.
+type: "rule"
+severity: "low"
+source: |
+  type.inbound
+  and any(attachments, .file_type in ("png", "jpg", "jpeg")
```

looks like jpeg is not actually a valid _file_type_ based on our enum

```suggestion
  and any(attachments, .file_type in ("png", "jpg")
```